ledger/participant-integration-api/src/test/suite/scala/platform/indexer/RecoveringIndexerSpec.scala

```diff
@@ -44,7 +44,7 @@ class RecoveringIndexerSpec
   private def readLog(): Seq[(Level, String)] = LogCollector.read[this.type, RecoveringIndexer]
 
   "RecoveringIndexer" should {
-    "work when the stream completes" in newLoggingContext { implicit logCtx =>
+    "work when the stream completes" in newLoggingContext { implicit loggingContext =>
```

Comment:
See if it makes sense to use `LoggingContext.ForTesting` here and remove a level of nesting.